examples/02_decoding/plot_haxby_grid_search.py

```diff
@@ -97,8 +97,7 @@
 anova_svc.fit(X, y)
 y_pred = anova_svc.predict(X)
 
-from sklearn.cross_validation import LeaveOneLabelOut, cross_val_score
-cv = LeaveOneLabelOut(session[session < 10])
```

Comment:
It used to be:
`cv = LeaveOneLabelOut(session[session < 10])`

However, in subsequent code, the variable `cv` was not used anywhere. With `model_selection`, it becomes `cv = LeaveOneGroupOut()` which has no context sensitive data to initialize it, making its use more nebulous. None of the comments indicated why it was being used.
I removed `cv` to trace its effect but the examples & tests remained unaffected. 

It seemed to me that it might be a relic.
I can't fiddle with the examples directly yet. @GaelVaroquaux will have to change this part, including the explanatory comments, or tell me exactly what to write here.

Reply:
the 'cross_val_score' function in line 109 could receive an argument `groups=session[session < 10]`